src/lib/sockets.ts

```diff
@@ -16,12 +16,15 @@
  */
 
 import Debug from "debug";
-import { GobanSocket, niceInterval, protocol } from "goban";
+import { GobanSocket, protocol } from "goban";
 
 const debug = new Debug("sockets");
 
 export const socket = new GobanSocket(window["websocket_host"] ?? window.location.origin);
 
+socket.options.ping_interval = 10000;
+socket.options.timeout_delay = 2000;
```

Comment:
Soo, depending on the day and the network weather, we can actually have folks who are using the site that have some fairly extreme latencies:

![image](https://github.com/online-go/online-go.com/assets/168460/929b295c-664b-47bc-8f2d-a49df5d8c090)

How about instead of `2000ms` we make that more dynamic? Something like, start it off at say 10s, and then scale it when we get a new latency to say either 2x or 3x their latency? That way we trigger sooner for those that have been on a very fast connection, and a lot slower for those that are across an ocean on a saturated link?

Reply:
Yeah - I had no data about what latencies we experience, but I did wonder about basing the timeout on latency like that.

Okie dokie

Reply:
Okie dokie